recipes/sqlite_zstd_vfs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sqlite_zstd_vfs" %}
+{% set version = "2024.01.13" %}
```

Comment:
Not sure what the best way is to version a git package

Reply:
If you expect this ever to get a version, prefix it with `0.0.`. Otherwise LGTM

Reply:
I'm not sure whether this will get an actual version, but I added a `0.0.` prefix in 6c997ea